src/test/app/claims/responseModelConverter.ts

```diff
@@ -40,7 +42,14 @@ describe('ResponseModelConverter', () => {
       const responseDraft = prepareResponseDraft(partyDetails)
       const responseData = prepareResponseData(party)
 
-      expect(ResponseModelConverter.convert(responseDraft)).to.deep.equal(responseData)
+      expect(ResponseModelConverter.fromDraft(responseDraft)).to.deep.equal(responseData)
     })
   })
+
+  it('converts part admission response', () => {
```

Comment:
We should at least test two main types of part admission. That test is pretty optimistic and conversion proved to be difficult as we had production issue due to incorrectly implemented conversion.